app/views/homepage/index.html.erb

```diff
@@ -57,17 +119,29 @@
       <div class="govuk-grid-row home-numbers-and-info">
         <h2 id="departments-and-policy-label" class="govuk-visually-hidden"><%= t('homepage.index.departments_and_policy_html') %></h2>
         <div class="govuk-grid-column-one-third">
-          <ul class="home-numbers">
+          <ul class="home-numbers" data-module="gem-track-click">
             <li>
-              <a href="/government/organisations#ministerial_departments" class="home-numbers__link home-numbers__link--first govuk-link">
+              <a href="/government/organisations#ministerial_departments"
+                 class="home-numbers__link home-numbers__link--first govuk-link"
+                 data-track-category="homepageClicked"
+                 data-track-action="departmentsLink"
+                 data-track-label="/government/organisations#ministerial_departments"
+                 data-track-dimension="<%= t('homepage.index.ministerial_departments_count') %> <%= t('homepage.index.ministerial_departments') %>"
```

Comment:
Oof. I've been picking away at a big number component in my spare time. Good catch here that I should add tracking options before I merge it and start deploying it across apps 😂